server/bin/pbench-restore-and-unpack-tarballs.py

```diff
@@ -88,6 +88,25 @@ def gen_list(backup):
                     yield tb_dt, c_entry.name, tb
 
 
+def escape_quotes(string):
+    """Escape single and double quotes in a string suitable for use in a shell
+    command.
+
+    Returns the string with the quotes escaped.
+    """
+    return re.sub("(['\"])", r"\\\1", string)
+
+
+def report_tb(*args, **kwargs):
```

Comment:
Since you don't use `args`, you can omit it from the signature (i.e., the following is legal):
```
def report_tb(**kwargs):
```
and I think that better reflects your intent (and, it forces all arguments to be keyword arguments, which I think is good for "safety").

Reply:
However, on the topic of "safety", it really would be better if the signature listed the required arguments -- if the caller omits or misspells one of them, that's going to be a runtime error which we won't find out about until the call is actually made (at which point the resulting `KeyError` will probably seem pretty mysterious).

Reply:
This method is a pretty thin passthrough; it's really not adding much but the format string (which could be a module level variable) and the `flush=True` vs just calling `print` directly. Seems you could at least make it pay for its supper by calculating `combined` itself rather than making each call point do it.